galaxy_ng/app/api/ui/viewsets/collection.py

```diff
@@ -136,7 +143,7 @@ def repo_filter(self, queryset, name, value):
 
     class Meta:
         model = CollectionVersion
-        fields = ['namespace', 'name', 'version', 'repository']
+        fields = ["namespace", "name", "version"]
```

Comment:
Removing `repository` entry since it implies an automated filter, but is overwritten by declared `repo_filter()` https://django-filter.readthedocs.io/en/latest/ref/filterset.html#declaring-filterable-fields